adele-booking/src/main/java/com/microservicesteam/adele/model/data/BookingDo.java

```diff
@@ -0,0 +1,74 @@
+package com.microservicesteam.adele.model.data;
+
+import static java.util.stream.Collectors.toList;
+import static javax.persistence.CascadeType.PERSIST;
+
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.OneToMany;
+
+import com.microservicesteam.adele.model.Booking;
+import com.microservicesteam.adele.model.PaymentStatus;
+
+@Entity
+public class BookingDo extends AbstractDo<Long> {
```

Comment:
You might hate me but I am not sure at the moment whether we want to persist `booking` the classical relational way. I think it worth to double check.